pkg/testrig/template/types.go

```diff
@@ -0,0 +1,24 @@
+package template
+
+type Vars struct {
+	Service Service `json:"service"`
+}
+
+type ServiceMap = map[string]Service
```

Comment:
type aliases (type A = B) should be used sparingly and usually only for migration of code amongst packages: https://golang.org/doc/go1.9#language

Can you just use a regular type here? :
```
type ServiceMap map[string]Service
```